pkg/plugins/resources/cargopackage/spec.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/updatecli/updatecli/pkg/plugins/utils/version"
 )
 
-// Spec defines a specification for a "dockerimage" resource
+// Spec defines a specification for a "cargopackage" resource
```

Comment:
Caught while reading code. Added to this (unrelated) PR as it is only a comment (e.g. no impact on code at all)